packages/embark/src/cmd/cmd_controller.js

```diff
@@ -403,15 +404,9 @@ class EmbarkController {
 
   }
 
-  reset() {
-    var fs = require('../lib/core/fs.js');
-    fs.removeSync('chains.json');
-    fs.removeSync('.embark/');
-    fs.removeSync('node_modules/.cache');
-    fs.removeSync('dist/');
-    fs.removeSync('embarkArtifacts/');
-    fs.removeSync('coverage/');
-    console.log(__("reset done!").green);
+  async reset() {
+    const doneMessage = __("reset done!").green;
+    await embarkReset({doneMessage});
```

Comment:
If the reset is still handled by Embark, why `embark-reset` needed in all the templates individually?

Reply:
It's because of the "wart" I mentioned in the PR description. When running `yarn reboot[:full]` sometimes `embark reset` would fail because `packages/embark/dist` wasn't already built. I had an `|| exit 0` in those scripts so that the reboot would continue regardless, but it's not ideal because the reset didn't actually happen. Also, I got the impression some people would see the error messages and `ctrl-c` the reboot thinking something was going wrong, and then they'd be left with a partially rebooted monorepo that might not reboot correctly without first manually deleting some `node_modules` directories.

So, all in all, the purpose is to address some subtle aspects of the monorepo, to keep development workflow smooth.